### PR TITLE
Layouts

### PR DESCRIPTION
**Pull request by shehlaMushtaqTkxel**

### CODE REVIEW

File: src/components/FilterComponent/index.tsx [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+    minWidth: 180,
+    color:
+      theme.palette.mode === "light"
+        ? "rgb(55, 65, 81)"
+        : theme.palette.grey[300],
+    boxShadow:
+      "rgb(255, 255, 255) 0px 0px 0px 0px, rgba(0, 0, 0, 0.05) 0px 0px 0px 1px, rgba(0, 0, 0, 0.1) 0px 10px 15px -3px, rgba(0, 0, 0, 0.05) 0px 4px 6px -2px",
+    "& .MuiMenu-list": {
+      "& .MuiMenu-list": {
+        width: "221pxpx",
```
Comment (mamoon-tkxel) [inline src/components/FilterComponent/index.tsx]: how it works (221pxpx) I don't know

File: src/components/FilterComponent/index.tsx [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+        onClose={handleClose}
+        sx={{ gap: "10px" }}
+      >
+        <MenuItem onClick={handleClose} disableRipple>
+          <SelectField
+            label="Type"
+            name="type"
+            className="select"
+            options={TYPE_OPTIONS}
+            sx={{
```
Comment (mamoon-tkxel) [inline src/components/FilterComponent/index.tsx]: add class and remove css from sx

File: src/components/FilterComponent/index.tsx [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+            //   value={values.deadlineDate}
+            //   onChange={handleChange}
+            name="deadlineDate"
+            label="Date"
+            type="date"
+            className="textField"
+            inputProps={{
+              min: getCurrentDate(),
+            }}
+            sx={{
```
Comment (mamoon-tkxel) [inline src/components/FilterComponent/index.tsx]: add class and remove css from sx

File: src/components/InputFields/index.tsx [hunk 4]
```diff
@@ -1,19 +1,27 @@
...
-      <Typography
+    <Stack position="relative">
+      <Stack gap={1}>
+        {label && <Typography>{label}</Typography>}
+        <TextField {...arg} error={Boolean(errorInputField)} />
+      </Stack>
+      {errorInputField && (  <Typography
         color="error"
-        className="error"
+        sx={{
```
Comment (mamoon-tkxel) [inline src/components/InputFields/index.tsx]: add class and remove css from sx

File: src/components/SelectField/index.tsx [hunk 5]
```diff
@@ -25,7 +25,19 @@ export const SelectField = ({name,label, className,value,errorSelect,options,onC
   return (
     <Stack gap="8px">
       {label && <Typography>{label}</Typography>}
-      <Select className={className} sx={{ minHeight: "44px" }} name={name} value={value} onChange={onChange}{...args}>
+      <Select 
+        className={className} 
+        sx={{
```
Comment (mamoon-tkxel) [inline src/components/SelectField/index.tsx]: add class and remove css from sx

File: src/index.css [hunk 6]
```diff
@@ -29,9 +29,9 @@
   margin-top: 4px;
 }
 .error{
```
Comment (mamoon-tkxel) [inline src/index.css:31]: remove commented code

File: src/views/admin/Grants/createGrant.tsx [hunk 7]
```diff
@@ -93,7 +93,6 @@ const CreateGrant = ({ initialValues }: CREATE_GRANT_TYPES) => {
                   options={TYPE_OPTIONS}
                   errorSelect={errors.type}
                 />
-                {errors.type && touched.type && errors.type}
```
Comment (mamoon-tkxel) [inline src/views/admin/Grants/createGrant.tsx:96]: why you have removed this line
{errors.type && touched.type && errors.type}

**Review by mamoon-tkxel — CHANGES_REQUESTED** [review]
plz check comments

File: src/views/admin/Grants/createGrant.tsx (hunk 7 above)
Comment (shehlaMushtaqTkxel) [reply src/views/admin/Grants/createGrant.tsx:96]: beacuase i am getting result without it

**Review by mamoon-tkxel — APPROVED** [review]

**Pull request merged**